cloud/grpc.go

```diff
@@ -70,23 +74,22 @@ func (c *Client) UnaryInterceptor(opts ...InterceptorOpt) grpc.UnaryClientInterc
 		}
 		ctx, err := c.reAuthIfExpired(ctx)
 		if err != nil {
-			return errors.Wrap(err, "failed refreshing expired token")
+			return errors.Wrapf(err, "failed refreshing expired token: %s", getReqID(ctx))
```

Comment:
nit: remove the `:`, otherwise it would be `failed refreshing expired token: 123: some-error-code`
```suggestion
			return errors.Wrapf(err, "failed refreshing expired token %s", getReqID(ctx))
```

Reply:
better yet, make it consistent with the change you made lower down in the code:

```
errors.Wrapf(err, "failed refreshing expired token {reqID: %s}", getReqID(ctx))
```